docs/50_hub.md

```diff
@@ -23,6 +23,22 @@ A "Base (MATLAB)" server is also available, which provides a MATLAB cloud instal
 
 If you need additional software installed in the image, you can add a server image that will be made available for all users in the `Server Options` menu.  Add a server image by updating the `profileList` in the [JupyterHub config file](https://github.com/dandi/dandi-hub/blob/do-eks/helm/jupyterhub/dandihub.yaml) and submitting a pull request to the [dandi-hub repository](https://github.com/dandi/dandi-hub).  Once the pull request is merged, the DANDI team will redeploy JupyterHub and the image will be available.
 
+See https://github.com/dandi/dandi-hub/pull/233 as an example of the changes you need.
+
+
+The following command will build and run a new image, which can be viewed locally in the browser at 127.0.0.1:8888/
+
+```sh
+docker build -f $(CONTAINERFILE) -t dandihub-dev:latest .
```

Comment:
```suggestion
docker build -f <Your-Dockerfile> -t dandihub-dev:latest .
```

Reply:
"lazy me" likes when I can just cut/paste commands.  Here if `CONTAINERFILE` variable was defined above, it would satisfy my desires -- it would also make it "easier" to customize since it is just that -- remove/place value.

Then here it should be

```suggestion
docker build -f "$(CONTAINERFILE)" -t dandihub-dev:latest .
```

to be kosher and no navigating relatively long line is needed to edit it etc.

Reply:
since to be used only once, defining a variable might be not really warranted though

Reply:
`Dockerfile.[gpu].<project-name>` - see other [Dockerfiles](https://github.com/dandi/dandi-hub/tree/main/images).